From 7362110778ebd8551a4f3d84ff962437cb530de9 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Fri, 29 Aug 2014 11:14:45 +0800
Subject: [PATCH 0084/1594] ENGR00329278-02 dtsi: imx6sl: evk: enable lcdif
 driver

commit d6cf2efad02182e9a46589a75a8abb6c146ae75a from
git://git.freescale.com/imx/linux-2.6-imx.git

-Enable lcdif driver
-Enable pwm backlight driver.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |    9 ++++++++-
 1 files changed, 8 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index b84dff2..a31f624 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -402,6 +402,12 @@
 			>;
 		};
 
+		pinctrl_pwm1_sleep: pwm1grp-sleep {
+			fsl,pins = <
+				MX6SL_PAD_PWM1__GPIO3_IO23 0x3080
+			>;
+		};
+
 		pinctrl_uart1: uart1grp {
 			fsl,pins = <
 				MX6SL_PAD_UART1_RXD__UART1_RX_DATA	0x1b0b1
@@ -577,8 +583,9 @@
 };
 
 &pwm1 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_pwm1>;
+	pinctrl-1 = <&pinctrl_pwm1_sleep>;
 	status = "okay";
 };
 
-- 
1.7.5.4

